ui/src/Components/notification/index.jsx

```diff
@@ -0,0 +1,81 @@
+import { useState } from "react";
+import { TYPETOCOLOR } from "../../Constant/constant";
+
+/**
+ * a pubsub kind of implementation, basically it will save a function in closure with varaible called "fn"
```

Comment:
Please rewrite the comments. Comments should not be using 'I'. It should be about what function is doing